frontend/src/ui-components/AcmTable/AcmTable.tsx

```diff
@@ -1162,8 +1217,45 @@ export function AcmTable<T>(props: AcmTableProps<T>) {
               <AcmManageColumn<T>
                 {...{ selectedColIds, setSelectedColIds, requiredColIds, defaultColIds, setColOrderIds, colOrderIds }}
                 allCols={columns.filter((col) => !col.isActionCol)}
+                toolbarItemSpacerSetting={showExportButton ? { default: 'spacerNone' } : undefined}
               />
             )}
+            {showExportButton && (
+              <ToolbarItem key={`export-toolbar-item`}>
+                <Dropdown
+                  onSelect={(event) => {
+                    event?.stopPropagation()
+                    setIsExportMenuOpen(false)
+                  }}
+                  className="export-dropdownMenu"
+                  toggle={
+                    <DropdownToggle
+                      toggleIndicator={null}
+                      onToggle={(value, event) => {
+                        event.stopPropagation()
+                        setIsExportMenuOpen(value)
+                      }}
+                      aria-label="export-search-result"
+                      id="export-search-result"
+                    >
+                      <ExportIcon />
+                    </DropdownToggle>
+                  }
+                  isOpen={isExportMenuOpen}
+                  isPlain
+                  dropdownItems={[
+                    <DropdownItem
+                      style={{ width: '10rem' }}
```

Comment:
What is this for? Seems to add space on the right of the "Export as CSV" text.

Reply:
It was for just that...we don't necessarily need it, so I've removed it but it was mostly to agree with the stylization happening on the search page export dropdown.